jupyter-docker/scripts/extension-installation/jupyter_install_combined_extension.sh

```diff
@@ -8,7 +8,7 @@ if [ -n "$1" ]; then
   mkdir ${JUPYTER_HOME}/${JUPYTER_EXTENSION_NAME}
   tar -xzf ${JUPYTER_EXTENSION} -C${JUPYTER_HOME}/${JUPYTER_EXTENSION_NAME}
   pip install -e ${JUPYTER_HOME}/${JUPYTER_EXTENSION_NAME}
-  jupyter serverextension enable --py ${JUPYTER_EXTENSION_NAME}
-  jupyter nbextension install --py ${JUPYTER_EXTENSION_NAME}
-  jupyter nbextension enable --py ${JUPYTER_EXTENSION_NAME}
+  sudo -u jupyter-user jupyter serverextension enable --py ${JUPYTER_EXTENSION_NAME}
```

Comment:
`tar` and `pip install` should be done as root, but `jupyter` commands should be done as jupyter-user. (If they are done as root they create root-owned jupyter config files, which causes issues.)